sdk/schemaregistry/schema-registry/src/generated/models/index.ts

```diff
@@ -45,17 +45,17 @@ export interface SchemaGetByIdHeaders {
 }
 
 /**
- * Defines headers for Schema_getIdByContent operation.
+ * Defines headers for Schema_queryIdByContent operation.
  */
-export interface SchemaGetIdByContentHeaders {
+export interface SchemaQueryIdByContentHeaders {
```

Comment:
Why are the generated models having the `Schema` prefix? I don't see anything in the swagger that would have caused this

Reply:
Only guessing from what I can see, but the operation IDs in the swagger of the `Schema_<Operation>`. The generator then groups all of the methods into `client.Schema.<Operation>` and then seems to generate names like `Schema<Operation>Headers`.  It is maybe not especially helpful for a REST surface as small as SchemaRegistry (currently just three operations to use operation groups like this, what other group will there be than `Schema` :) I definitely did not want this indirection from client to one and only one thing to three methods in the public API. I think if the Schema_ were dropped from the swagger, the API would come out flatter without this prefix. I think using Schema_ was actually suggested on the swagger PR, though.